NickvisionTagger.Shared/Controllers/MainWindowController.cs

```diff
@@ -385,6 +385,7 @@ public void UpdateTags(PropertyMap map, bool triggerSelectedMusicFilesProperties
                 try
                 {
                     pair.Value.Filename = map.Filename;
+                    map.Filename = pair.Value.Filename;
```

Comment:
Is this supposed to swap the two values?

Reply:
Because the settings MusicFile's filename has the possibility of changing the characters (replacing invalid ones) we set the mpa value again to provide the possibly new version to use for showing the UI.

Reply:
Right, the FIlename setter 😅